tavern/_plugins/mqtt/client.py

```diff
@@ -155,7 +155,7 @@ def __init__(self, **kwargs):
 
         self._client.on_message = self._on_message
 
-        if self._tls_args:
```

Comment:
Instead of returning an extra value from `_handle_tls_args`, couldn't this just be `if self._tls args is not None`?